src/SendHeaders.php

```diff
@@ -40,7 +40,7 @@ public function setCacheHeader(): void
 		if (count($this->pageProperties)) {
 			// if we are logged in, or on the front page we don't need to worry about configuring things further
 			if (
-				($this->pageProperties['isLoggedInUser'] || $this->pageProperties['requiresPassword'])
+				($this->pageProperties['isLoggedInUser'] || $this->pageProperties['requiresPassword'] || $this->pageProperties['isPreviewPage'])
 				&& !$this->developerMode
```

Comment:
Suggestion: remove the `developerMode` check here altogether, so there's no risk of developer mode interfering with cases where a `no-cache` header should be sent, regardless of what the environment type is.

Reply:
We should probably refactor to decouple the developer mode output from the core behaviour of the plugin at some point. But until then I think it's preferable to play it safe in terms of ensuring that pages that shouldn't be publicly available don't get cached.